src/lib/utils/thorchain/memo/addLimitToMemo.ts

```diff
@@ -0,0 +1,64 @@
+import { assertAndProcessMemo } from './assertAndProcessMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddLimitToMemoArgs = {
+  // A THORChain memo, with or without a limit/MinOut
+  memo: string
+  // The limit/minout to be added to be added to the memo. Must be in THOR base unit, or problems.
+  limit: string
+}
+
+// A simple util that adds a limit to a THORChain memo, if the memo doesn't already have one.
+// This is made to be composable, do one thing, and do it well.
+// If you need to handle slippage concerns, this should be done beforehand, with addLimitToMemo/assertAndProcessMemo being the final steps.
+export const addLimitToMemo = ({ memo, limit }: AddLimitToMemoArgs) => {
```

Comment:
If we use regular old function params instead of kwargs we can label them with docstring instead of 2 layers of indirection via a custom type and code comment

Reply:
I'm not sure about good ol' function params here, potato potato and comments in types *should* be reflected.
Except they were not proper JSDoc format comments! Apparently multiline is the way, and obviously for the comment above `addLimitToMemo` too:

<img width="695" alt="image" src="https://github.com/shapeshift/web/assets/17035424/62001bfa-c722-44df-9fa5-f01a41f7b0ac">

Reply:
57645ea765

<img width="1320" alt="image" src="https://github.com/shapeshift/web/assets/17035424/d8aef31c-34fa-4827-9e9b-98ccf53f50f6">
<img width="1337" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3e0e1119-28dc-434f-b0d9-38f69b38d6e6">